### PR TITLE
try to run some tests on GPU

### PR DESCRIPTION
**Pull request by trontrytel**

### CODE REVIEW

**Comment by trontrytel** [comment]
bors try

**Comment by trontrytel** [comment]
bors try

**Comment by trontrytel** [comment]
bors try

**Comment by trontrytel** [comment]
bors try

File: test/gpu_tests.jl [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+        )
+
+        output[5, i] = CM.AerosolActivation.total_M_activated(
+            param_set,
+            aerosol_distr_B,
+            T,
+            p,
+            w,
+            q,
+        )
```
Comment (charleskawczynski) [inline test/gpu_tests.jl:118]:
```suggestion
        args = (T, p, w, q)
        output[3, i] = CM.AerosolActivation.total_N_activated(param_set, aerosol_distr_B, args...)
        output[4, i] = CM.AerosolActivation.total_N_activated(param_set, aerosol_distr_κ, args...)
        output[5, i] = CM.AerosolActivation.total_M_activated(param_set, aerosol_distr_B, args...)
```

Can you reduce number of lines with this? (I haven't tried against the formatter)

Comment (charleskawczynski) [reply test/gpu_tests.jl:118]: Oh, and you could apply to `output[6, i]`, too

File: test/gpu_tests.jl [hunk 2]
```diff
@@ -0,0 +1,373 @@
+using Test
+using KernelAbstractions
+using CUDAKernels
+
+import CloudMicrophysics
+import CLIMAParameters
+import Thermodynamics
+
+const CM = CloudMicrophysics
```
Comment (charleskawczynski) [inline test/gpu_tests.jl:9]:
```suggestion
const CM = CloudMicrophysics
const CM0M = CloudMicrophysics.Microphysics_0M
const CM1M = CloudMicrophysics.Microphysics_1M
```
I might add these aliases to see if you could squeeze out a few more lines of code.

**Review by charleskawczynski — APPROVED** [review]
Overall, I think this looks good/correct from the code point of view.

**Comment by trontrytel** [comment]
> Overall, I think this looks good/correct from the code point of view.

Thanks! I'll add your suggestions and see if can make this shorter. And will let you know if buildkite finds any GPU issues...

**Comment by trontrytel** [comment]
This PR solves https://github.com/CliMA/CloudMicrophysics.jl/issues/40

**Pull request merged**